test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -51,6 +51,13 @@ end
     end
 end
 
+@testset "Quadratic tests" begin
+    MOIT.contquadratictest(
+        SOLVER,
+        MOIT.TestConfig(atol=1e-3, rtol=1e-3, duals=false, query=false)
```

Comment:
Since you've hooked up duals and the get quadratic constraint, I think you can have `duals=true, query=true`

Reply:
Now I need to implement `LQOI.get_quadratic_terms_objective` to remove 5 errors in tests.